fineract-provider/config/checkstyle/checkstyle.xml

```diff
@@ -35,10 +35,10 @@
         <property name="format" value="\s+$"/>
         <property name="message" value="Line has trailing spaces."/>
     </module>
-    <module name="RegexpMultiline">
+    <!-- <module name="RegexpMultiline">
```

Comment:
why comment this out here for making this change?

Reply:
Oh I think you missed it, I wrote a note above that I will remove that while I remove the comments.